src/libpgmoneta/configuration.c

```diff
@@ -158,20 +159,21 @@ pgmoneta_read_configuration(void* shm, char* filename)
 
    while (fgets(line, sizeof(line), file))
    {
-      if (!is_empty_string(line))
+      parsed_line = pgmoneta_remove_whitespace(line);
```

Comment:
You are missing `free(parsed_line)` for all paths, so you need to move it.

Use valgrind for checking

Reply:
yeah sure i will check now